open_buildings/settings.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+from typing import Dict
+from pydantic import BaseModel
+
+class Source(Enum):
+    GOOGLE = 1
+    OVERTURE = 2
+
+class Format(Enum):
+    SHAPEFILE = 1
+    GEOJSON = 2
+    GEOPACKAGE = 3
+    FLATGEOBUF = 4
+    PARQUET = 5
+
+
+class SourceSettings(BaseModel):
+    base_url: str
+    hive_partitioning: bool
+
+class SettingsSchema(BaseModel):
+    sources: Dict[Source, SourceSettings]
+    extensions: Dict[Format, str]
+
+settings = SettingsSchema(
+    sources={
+        Source.GOOGLE: SourceSettings(
+            base_url="s3://us-west-2.opendata.source.coop/google-research-open-buildings/geoparquet-by-country/*/*.parquet",
+            hive_partitioning=True
+        ),
+        Source.OVERTURE: SourceSettings(
+            base_url="s3://us-west-2.opendata.source.coop/cholmes/overture/geoparquet-country-quad-hive/*/*.parquet",
+            hive_partitioning=True
+        )
+    },
+    extensions={
+        Format.SHAPEFILE: 'shp',
+        Format.GEOJSON: 'json',
```

Comment:
Is there a way to have this recognize both .geojson and .json as 'GeoJSON'? My previous solution was a total hack:

```
    output_extension = {
        'shapefile': '.shp',
        'geojson': 'json',
```

It just checked for ending in 'json' so it'd grab both .json and .geojson. Tagged it in #49 - don't need more in this PR, and would love to get this one in asap to start to have a test framework.

Reply:
oh good point!